test/v1alpha1_samples/catalog.go

```diff
@@ -225,6 +225,9 @@ func (c *Catalog) BuildWithOutputSecret(name string, ns string, secretName strin
 // ClusterBuildStrategy to support tests
 func (c *Catalog) ClusterBuildStrategy(name string) *build.ClusterBuildStrategy {
 	return &build.ClusterBuildStrategy{
+		TypeMeta: metav1.TypeMeta{
+			Kind: string(build.ClusterBuildStrategyKind),
+		},
```

Comment:
Using `ClusterBuildStrategy` caused issues since the BuildRuns created using this strategy in the test code did not have the kind and therefore failed to be usable.